### ISSUE TITLE
Warning message when using 'Use Current Page' (but still adding BBC programme to queue)

### ISSUE THREAD

**Issue by jefro108** [Ascoware/get-iplayer-automator#324]
Using the latest version downloaded today (thanks!) when I click on 'Use Current Page' on https://www.bbc.co.uk/programmes/p08mlg6x I get a warning message. See screenshot
<img width="1093" alt="Screenshot 2021-04-07 at 22 32 12" src="https://user-images.githubusercontent.com/12976435/113939712-3b70e180-97f4-11eb-9724-a06a5d3ca2b6.png">

Strangely it thinks it's a series, but it's a single programme 'clip', which it goes ahead to add to the queue anyway!

Are you aware of this anomaly?

**Comment by skovatch** [comment]
Nope, first time I've seen it. Odd that it adds it anyway; I thought it would have bailed after the alert appeared.

**Comment by jefro108** [comment]
I'm wondering if it's something to do with it being categorised as a 'clip' rather than a programme - since the Alt History series that it is part of - and is what it comes up in GiA as - doesn't list it as an episode - see https://www.bbc.co.uk/programmes/p07dkx93/episodes/player but instead it is in this list https://www.bbc.co.uk/programmes/p07dkx93/clips
Even though it conforms to the same url scheme as an episode like https://www.bbc.co.uk/programmes/p07dlcpp presumably get-iplayer gets confused?
